Document bluetooth proxy support (#46472)
home-assistant/home-assistant.github.io: source/_integrations/smlight.markdown

#### Infrared

- **IR Emitter**: This entity can be used by other integrations as an [Infrared](/integrations/infrared/) proxy to send IR commands through the SLZB-Ultima device. For example, you can use the [LG Infrared](/integrations/lg_infrared/) integration with this entity type to control your TV.

> ## Bluetooth proxy

Reviewer: ```suggestion
## Bluetooth remote adapter (proxy)
```

Reviewer: It was called remote adapter here, https://github.com/OpenHomeFoundation/roadmap/issues/188, so I'm renaming it to stay consistent.

> SMLIGHT SLZB U-series devices running SLZB-OS can act as a Bluetooth proxy. They scan for and forward Bluetooth advertisement data to Home Assistant.

Reviewer: ```suggestion
SMLIGHT SLZB U-series devices running SLZB-OS can act as a Bluetooth remote adapter (proxy). They scan for and forward Bluetooth advertisement data to Home Assistant.
```

> To use the Bluetooth proxy, you must enable the Bluetooth Low Energy (BLE) proxy in the SMLIGHT device's web UI. Once enabled, Home Assistant automatically receives the forwarded Bluetooth advertisement data; no additional configuration is required in Home Assistant.

Reviewer: ```suggestion
To use the Bluetooth remote adapter, you must enable the Bluetooth Low Energy (BLE) remote adapter in the SMLIGHT device's web UI. Once enabled, Home Assistant automatically receives the forwarded Bluetooth advertisement data. No additional configuration is required in Home Assistant.
```
